usr/share/rear/rescue/GNU/Linux/310_network_devices.sh

```diff
@@ -854,7 +854,11 @@ function handle_physdev () {
 
     DebugPrint "$network_interface is a physical device"
 
-    mac="$( cat $sysfspath/address )" || BugError "Could not read a MAC address for '$network_interface'."
+    if [ -e $sysfspath/bonding_slave/perm_hwaddr ] ; then
+        mac="$( cat $sysfspath/bonding_slave/perm_hwaddr )"
```

Comment:
This fixes the issue for bonding, but an analogous problem exists for teaming. As a result, teaming with more than one team member is broken (unless one uses SIMPLIFY_TEAMING, but this does not work with LACP).

Reply:
Shouldn't one use ethtool -P always instead of a bonding-specific fix?

Reply:
You are right, the issue is still there for Teaming.
Checking the code, the only reliable way seems to be using `ethtool -P`.
Having the `ethtool` program installed is a must.

Reply:
Due to #1934 being applied, those with multiple interfaces in a team will now suffer from a regression. I think SIMPLIFY_TEAMING=yes should remain the default for now.

Reply:
See PR https://github.com/rear/rear/pull/2065